appveyor.yml

```diff
@@ -0,0 +1,99 @@
+build: false
+
+clone_folder: C:\projects\pydm
+
+platform:
+  #- x86
+  - x64
+
+environment:
+  PYQT_VERSION: 5
+  OFFICIAL_REPO: "slaclab/pydm"
+  ANACONDA_PYDM_DEV:
+    secure: ccW0d6nCipLq5cps11ptt16klr2qn8bYhv7WX5pZ3d+WaC1QbXs6q+Pl9RPa578L
+  ANACONDA_PYDM_TAG:
+    secure: qROm2iOANaz9zZRZzyde8b7SMNpEC3rDP78SLAGaoRZBrLiE1MzP62VLjqfXMYBN
+
+  matrix:
+    - PYTHON_VERSION: 2.7
+      MINICONDA: C:\Miniconda
+    - PYTHON_VERSION: 3.5
+      MINICONDA: C:\Miniconda35
+    - PYTHON_VERSION: 3.6
+      MINICONDA: C:\Miniconda36
+
+
+init:
+  - ps: if ($Env:PLATFORM -eq "x64") { $Env:MINICONDA = "${Env:MINICONDA}-x64" }
+  - ps: Write-Host $Env:PYTHON_VERSION
+  - ps: Write-Host $Env:MINICONDA
+  - ps: Write-Host $Env:GITHUB_REPO_NAME
+  - ps: Write-Host $Env:PLATFORM
+  - ps: Write-Host $Env:APPVEYOR_REPO_TAG
+  - ps: Write-Host $Env:APPVEYOR_REPO_TAG_NAME
+  - ps: Write-Host $Env:APPVEYOR_REPO_NAME
+  - ps: Write-Host $Env:APPVEYOR_PULL_REQUEST_NUMBER
+  - ps: Write-Host $Env:APPVEYOR_REPO_BRANCH
+
+install:
+  - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%PATH%"
+  - conda config --set always_yes yes
+  - conda install conda-build anaconda-client
+  - conda update -q conda conda-build
+  # Add the channels needed for install
+  - conda config --append channels conda-forge
+  # Build the recipe and use it the local build for install
+  - "conda build -q conda-recipe --python=%TRAVIS_PYTHON_VERSION% --output-folder bld-dir"
+  - "conda config --add channels \"file:///C:/projects/pydm/bld-dir\""
+  - "conda create -q -n test-environment python=%TRAVIS_PYTHON_VERSION% pydm --file dev-requirements.txt"
+  - activate test-environment
+
+test_script:
+  - python run_tests.py
+
+after_test:
+  - codecov
```

Comment:
Do you want both `appveyor` and `Travis` to report code coverage? I might just pick one...

Reply:
Take a look here: https://github.com/codecov/support/wiki/Merging-Reports

Reply:
Nice. Just wasn't sure that `codecov` differentiated the two